packages/studioCMS/src/integrations/studioCMSDashboard/routes/authroutes/components/OauthProviders.astro

```diff
@@ -0,0 +1,108 @@
+---
+import { urlGenFactory } from 'studiocms:helpers';
+import { authEnvCheck } from "studiocms-dashboard:auth";
+import type Config from 'virtual:studiocms/config';
+
+type Props = {
+    dashboardConfig: typeof Config.dashboardConfig;
+};
+
+const { 
+    dashboardConfig: {
+        AuthConfig: {
+            providers
+        },
+        dashboardRouteOverride
+    },
+} = Astro.props;
+
+const githubLoginURL = await urlGenFactory(true, 'login/github', dashboardRouteOverride)
+const discordLoginURL = await urlGenFactory(true, 'login/discord', dashboardRouteOverride)
+const googleLoginURL = await urlGenFactory(true, 'login/google', dashboardRouteOverride)
+const auth0LoginURL = await urlGenFactory(true, 'login/auth0', dashboardRouteOverride)
+
+const { 
+    DISCORD: {
+        ENABLED: discordEnabled
+    },
+    GITHUB: {
+        ENABLED: githubEnabled
+    },
+    GOOGLE: {
+        ENABLED: googleEnabled
+    },
+    AUTH0: {
+        ENABLED: auth0Enabled
+    },
+    SHOW_OAUTH,
+    SHOW_PROVIDER_ERROR
+ } = await authEnvCheck(providers);
+
+---
+{SHOW_OAUTH && ( 
+    <center>
```

Comment:
pls change to a supported tag

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center